src/Components/DayLine/DayLine.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from "react";
+// Components
+// Context
+// Hooks
+// Pages
+// Resources
+
+export interface Props {
+    date: Date;
+}
+
+const DayLine: FC<Props> = ({ date }: Props) => {
+    const dataDay = () => {
+        return Math.floor((new Date().getTime() - date.getTime()) / (1000 * 3600 * 24));
+    };
```

Comment:
Este código parece-me bem, mas tenta fazer o mesmo com o momentjs.

Reply:
Ele não deveria mostrar a linha se a data for inferior a 1 dia:
![image](https://user-images.githubusercontent.com/32296976/117048702-cca77b00-ad02-11eb-82ed-7f77c3292387.png)